pandas/tests/copy_view/test_clip.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+from pandas import DataFrame
+import pandas._testing as tm
+from pandas.tests.copy_view.util import get_array
+
+
+def test_clip_inplace_reference(using_copy_on_write):
+    df = DataFrame({"a": [1.5, 2, 3]})
+    df_copy = df.copy()
+    arr_a = get_array(df, "a")
+    view = df[:]
+    df.clip(lower=2, inplace=True)
+
+    # Clip not actually inplace right now but could be
```

Comment:
Can you do: TODO(CoW): ...

Reply:
Ah no, could you add another test without a reference? I think it should pass after my where pr is in

Reply:
I think it is not actually inplace even without CoW.

Reply:
Yep you are correct. We should refactor this if we decide to keep inplace here